Directory.Build.props

```diff
@@ -47,6 +47,10 @@
 		<ContinuousIntegrationBuild>true</ContinuousIntegrationBuild>
 	</PropertyGroup>
 
+  <PropertyGroup Condition="!$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net8.0'))">
```

Comment:
I'm a bit confused. Are you trying to say "if you don't target net8.0, then use the 8.0.4 System.Text.Json package"?

Note that you don't need to explicitly reference the System.Text.Json package if you target `net8.0`. It is available automatically.

Reply:
Sorry - typo for the `!` - I wanted to include System.Text.Json for net8+.
As you mentioned offline, I can reference System.Text.Json for all the target frameworks so I've updated to that.